Unity-Technologies/com.unity.multiplayer.samples.bitesize
# [MTT-9236] Fix shader warnings through renaming

### Description


Fixed the leftover warnings regarding the shader through renaming ToonLightingModel subgraph.

Test if warning disappeared on your machine:
- delete library of DistributedAuthoritySample project
- open project freshy
- check if any warnings come up in the editor

### Issue Number(s)


https://jira.unity3d.com/browse/MTT-9236

### Contribution checklist
